### PR TITLE
Fix multi_get

### PR DESCRIPTION
**Pull request by mikhailOK**
Make multi_get return Vec<Result> instead of Result<Vec>.
See #502 for details.

### CODE REVIEW

File: src/db.rs [hunk 1]
```diff
@@ -1779,16 +1783,25 @@ fn convert_options(opts: &[(&str, &str)]) -> Result<Vec<(CString, CString)>, Err
         .collect()
 }
 
-fn convert_values(values: Vec<*mut c_char>, values_sizes: Vec<usize>) -> Vec<Vec<u8>> {
+fn convert_values(
+    values: Vec<*mut c_char>,
+    values_sizes: Vec<usize>,
+    errors: Vec<*mut c_char>,
+) -> Vec<Result<Option<Vec<u8>>, Error>> {
```
Comment (stanislav-tkach) [inline src/db.rs:1790]: Do you think that `Option<Vec>` would be better than a potentially empty `Vec` here?

Personally, I would use `Ok(value.unwrap_or(Vec::new()))` to simplify api a little bit.

**Review by stanislav-tkach — APPROVED** [review]
Thank you for the pull request! It is nice that you reuse the `ffi_util::raw_data` function here, though I'm not sure about `Option<Vec>`.

File: src/db.rs (hunk 1 above)
Comment (mikhailOK) [reply src/db.rs:1790]: Without `Option` there's no way to distinguish between key -> empty string and key not in the database. This way `multi_get` is consistent with `get` that returns `Result<Option<Vec>>`.

**Comment by thibault-martinez** [comment]
Hello, we're currently having a `signal: 11, SIGSEGV: invalid memory reference` using `multi_get` and it may be due to #502 , do you have an idea when you intend to merge this PR ? Thanks.

**Review by aleksuss — APPROVED** [review]

**Comment by thibault-martinez** [comment]
FYI, it did fix out SIGSEGV, thanks!

**Pull request merged**